docs/app/content-items.md

```diff
@@ -1,98 +1,188 @@
 
-This feature is only available if an Archive column has been configured in the Collection's data model settings.
+This action permanently removes the Item from the database. Proceed with caution!
 
 :::
 
-::: tip Archiving from the Item Detail page
+:::tip Archive as a Soft-Delete
 
-You can also click on an item to navigate to its Detail page. From there, you can click on the "Archive Item" (archive
-icon) button located in the header.
+If you wish to remove an Item from view or use, but keep it stored, you may consider
+[archiving the Item](#archive-an-item) instead.
 
 :::
 
-## Reverting an Item
+## Batch Delete, Archive or Edit Items
+
+<video autoplay muted loop controls title="Batch Edit Items">
+	<source src="" type="video/mp4" />
+</video>
+
+It is possible to delete, archive, or edit multiple Items at once from the Collection Page. Deletion will permanently
+delete Items from the database, this cannot be reverted. When batch editing, the Item Page opens, allowing you to make
+the same edit across multiple Items. Follow the steps below to batch delete, archive, or edit Items.
 
-1. Navigate to the [Content Module](/app/overview#1.-module-bar)
-2. Navigate to the collection of the item you want to archive
-3. Click on an item to navigate to its Detail page
-4. Click "Revisions" in the page sidebar
-5. Click on the revision you wish to preview
-6. Click on the "Revert" button located in the drawer's header
-7. Confirm the action within the dialog by clicking "Revert"
+1. Navigate to the Content Module and select the desired Collection.
+2. Choose a [Layout](/#adjust-item-layouts) that supports multiple Item selection from the Sidebar.\
+   (e.g. The Table or Card Layout)
+3. Click the Items you wish to select:
+   - **Table Layout:** Click <span mi icon>check_box_outline_blank</span> in the left-most column.
+   - **Card Layout:** Click <span mi icon>radio_button_unchecked</span> in the upper left hand corner of each Card.
 
-## Commenting on an Item
+Once Items are selected, the following icons will appear in the Page Header: <span mi btn muted>delete</span>
+<span mi btn muted>archive</span> <span mi btn muted>edit</span>
 
-1. Navigate to the [Content Module](/app/overview#1.-module-bar)
-2. Navigate to the collection of the item you want to comment on
-3. Click on an item to navigate to its Detail page
-4. Click "Comments" in the page sidebar
-5. Type your message within the input
-6. Click the "Submit" button
+**Delete**\
+4. Click <span mi btn dngr>delete</span> in the header and a popup will appear.\
+5. Click **"Delete"** to confirm the action.
 
-### Mentions
+**Archive**\
+4. Click <span mi btn muted>archive</span> in the header and a popup will appear.\
+5. Click **"Delete"** to confirm the action.
+
+**Edit**\
+4. Click <span mi btn muted>edit</span> in the header and the Item Page will open.\
+5. Update Item details as desired.\
+6. Click <span mi btn>check</span> to confirm batch edits on all Items.
+
+:::danger DANGER!
```

Comment:
Remove this `DANGER!` text.

Reply:
[Danger! High Voltage](https://open.spotify.com/track/2VHfyWLlvCvbkIqtS5tqt3?si=a176a059c00e4960) ⚡ ☠️